tests/core/test_decorator.py

```diff
@@ -0,0 +1,31 @@
+
+import pytest
+from fastats.core.decorator import fs
+
+
+@pytest.mark.xfail
```

Comment:
I really like the `xfail` markings, we definitely need more of these, as everyone tends to always test the success cases.